### PR TITLE
[PRM] LRSD-4764 Add columns for Start/End Activity Period

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

**Comment by claricebrasil** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
ci:test:sf

**Review by eliasantosf — APPROVED** [review]

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/Table/Table.tsx [hunk 1]
```diff
@@ -122,12 +122,13 @@ const Table = <T extends BasicRow>({
 			CLOSE_DATE: 'closeDate',
 			DATE_SUBMITTED: 'submitDate',
 			DEAL_DATE_SUBMITTED: 'dateCreated',
+			END_ACT_PERIOD: 'maxDateActivity',
 			OPPORTUNITY_ACCOUNT_NAME: 'accountName',
 			PARTNER: 'companyName',
 			PARTNER_ACCOUNT_NAME: 'partnerAccountName',
 			PARTNER_NAME: 'partnerFirstName',
 			REQUEST_STATUS: 'mdfRequestStatus',
-			STAGE: 'stage',
```
[the remark this answers is not in the document]
Comment (claricebrasil) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/Table/Table.tsx:130]: Done

**Comment by claricebrasil** [comment]
ci:test:sf

**Review by ryanschuhler — APPROVED** [review]

**Comment by ryanschuhler** [comment]
ci:forward:force

**Comment by claricebrasil** [comment]
resent: https://github.com/brianchandotcom/liferay-portal/pull/151403

**Pull request closed without merging**